contrib/platform/test/com/sun/jna/platform/unix/LibCTest.java

```diff
@@ -79,10 +84,10 @@ public void testGethostnameGetdomainname() {
         LibC.INSTANCE.gethostname(buffer, buffer.length);
         String hostname = Native.toString(buffer);
         System.out.println("Hostname: " + hostname);
-        assertTrue(hostname.length() > 0);
+        assertNotNull(hostname.length() > 0);
```

Comment:
This change doesn't seem right (also 4 lines below)

Reply:
You are right - the idea was to replace the length check with a not-null check. The AIX system on the gcc buildfarm has the FQDN as hostname and an empty domain name.